ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -370,6 +370,8 @@ EFVSC:
   - exhaust_fan_speed_percentage_sensor
   - exhaust_fan_current_sensor
   - exhaust_fan_power_sensor
+  - exhaust_fan_run_mode
```

Comment:
probably need to move to VSC based on comments below. remove exhaust fan from field.